test_drupal_quicksprint.sh

```diff
@@ -15,12 +15,15 @@ tar -C /tmp -zxf ~/tmp/drupal_sprint_package.no_docker.$(cat ./.quicksprint_rele
 # Run the standard install_ddev.sh, asserting we already have docker set up.
 cd /tmp/drupal_sprint_package && echo y | ./install_ddev.sh
 
+
 # Remove any existing sprints, then do start_sprint.sh
-cd ~/Sites/sprint && chmod -R u+w sprint-* && rm -rf sprint-* echo y | ./start_sprint.sh
+cd ~/Sites/sprint
+find ~/Sites/sprint -type d -name "sprint-2*" -exec chmod -R u+w '{}' \; -exec rm -rf '{}' \;
+echo y | ./start_sprint.sh
 
 # Run start_sprint in the (only) sprint directory
 cd ~/Sites/sprint/sprint-2* && echo y | ./start_clean.sh
```

Comment:
There's probably some exit 0/1 assertion that can be done afterward.

Maybe check ddev health again?